### PR TITLE
Karma doesn't need webpack entry points in the config

### PR DESCRIPTION
**Pull request by greypants**

### CODE REVIEW

File: gulpfile.js/config/webpack.js [hunk 1]
```diff
@@ -38,6 +27,18 @@ module.exports = function(env) {
...
+    webpackConfig.entry = {
+      page1: [jsSrc + 'page1.js'],
+      page2: [jsSrc + 'page2.js']
+    }
+
+    webpackConfig.output = {
+      path: jsDest,
+      filename: env === 'production' ? '[name]-[hash].js' : '[name].js',
+      publicPath: publicPath
+    }
```
Comment (nhunzaker) [inline gulpfile.js/config/webpack.js]: Is output necessary in addition to entries?

Comment (greypants) [inline gulpfile.js/config/webpack.js]: What do you mean?

Comment (nhunzaker) [inline gulpfile.js/config/webpack.js]: Do you need to avoid the output setting in test mode. Will it throw a similar error?

Comment (greypants) [inline gulpfile.js/config/webpack.js]: Oh, I dunno that it'll specifically hurt to leave it. But whatever webpack defaults to is fine, so I figured I'd only add this custom stuff for the environments that need it.

Comment (nhunzaker) [inline gulpfile.js/config/webpack.js]: Cool.

**Pull request merged**